src/Elmish/Foreign.purs

```diff
@@ -195,6 +196,10 @@ instance CanReceiveFromJavaScript a => CanReceiveFromJavaScript (Nullable a) whe
             Valid -> Valid
             Invalid err -> Invalid err { expected = "Nullable " <> err.expected }
 
+instance CanPassToJavaScript a => CanPassToJavaScript (Opt a)
+instance CanReceiveFromJavaScript a => CanReceiveFromJavaScript (Opt a) where
+    validateForeignType _ v = validateForeignType (Proxy :: _ (Nullable a)) v
```

Comment:
Does `null` get parsed as `undefined`?

Reply:
Good question. I will investigate.

Reply:
In `Nullable` it does as it’s using `==` equality: https://github.com/purescript-contrib/purescript-nullable/blob/8b19c16b16593102ae5d5d9f5b42eea0e213e2f5/src/Data/Nullable.js#L8